exasol_script_languages_developer_sandbox/lib/config.py

```diff
@@ -1,6 +1,8 @@
 
 default_config = {
     "time_to_wait_for_polling": 10.0,
+    # Source AMI is set to Ubuntu 20.04
+    "source_ami_id": "ami-0c9354388bb36c088"
```

Comment:
ami ids are region and version specific, I think, AWS also remove older AMIs of Ubuntu after a while, because they miss to many updates.

Reply:
I opened an issue: #26